examples/sleep/src/test/java/SleepTest.java

```diff
@@ -0,0 +1,10 @@
+import static org.junit.Assert.assertNotNull;
+import org.junit.Test;
+
+public class SleepTest {
```

Comment:
I wonder the value of these tests? These are only example programs and the tests aren't really exercising the behavior of the classes apart from whatever is in the constructor.